meta_cuda.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I didn't mind so so much when there was just one `meta.json` at the root of the project, but I personally dislike cluttered root directories in projects. Is there a way we could push these down a level? Maybe into `etc/viam` or somewhere?